rclpy/rclpy/impl/implementation_singleton.py

```diff
@@ -38,3 +38,4 @@
             " Please refer to '%s' for possible solutions" % \
             (e.path, 'https://github.com/ros2/ros2/wiki/Rclpy-Import-error-hint')
     raise
+rclpy_logging_implementation = importlib.import_module('._rclpy_logging', package='rclpy')
```

Comment:
Should this have the same `ImportError` check as the other library?

Reply:
yes, I wasn't thinking of the cases where _rclpy imports fine but _rclpy_logging has issues. 2ad25a8b68520f7d97c822633bedff1e05e525f6